### PR TITLE
#29 Create app footer

### PR DESCRIPTION
**Pull request by YarkoSumyk**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (Alan18081) [inline .env:1]: Please, remove from commit `.env` file

File: .env.example [hunk 2]
```diff
@@ -1,4 +0,0 @@
-SKIP_PREFLIGHT_CHECK=true
```
Comment (Alan18081) [inline .env.example:1]: Please, don't remove this file, it is an example

File: src/components/App-footer/app-footer.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { useStyles } from './app-footer.styles';
+
+import FooterLists from '../../containers/footer-lists';
+import FooterLinks from '../../containers/footer-links';
+
+const AppFooter = () => {
+  const classes = useStyles();
```
Comment (Alan18081) [inline src/components/App-footer/app-footer.js:9]: Please, use `styles` instead of `classes`

File: src/components/Cabinet/Cabinet.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import NotLoggedCabinet from '../NotLoggedCabinet';
+import LoggedCabinet from '../LoggedCabinet';
+
+const Cabinet = () => {
+  const isLogged = false;
+  const props = {
+    logged: isLogged
+  };
+
+  const classes = useStyles(props);
```
Comment (Alan18081) [inline src/components/Cabinet/Cabinet.js:14]: Please, use `styles` instead

File: src/components/LoggedCabinet/LoggedCabinet.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+import { useStyles } from './LoggedCabinet.styles';
+import { setThemeMode } from '../../redux/theme/theme.actions';
+import { LANGUAGE, CABINET_OPTIONS_LOGGED } from '../../configs';
+
+const LoggedCabinet = () => {
+  const lightMode = useSelector(({ Theme }) => Theme.lightMode);
+  const dispatch = useDispatch();
+
+  const classes = useStyles();
```
Comment (Alan18081) [inline src/components/LoggedCabinet/LoggedCabinet.js:18]: Please, use name `styles` instead

File: src/containers/footer-lists/footer-lists.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        </a>
+      </Typography>
+    </div>
+  ));
+  return (
+    <div className={styles.cardDeck}>
+      <div className={styles.cardBody}>
+        <div className={styles.cardTitle}>
+          <Typography variant='h5'>
+            {' '}
```
Comment (Alan18081) [inline src/containers/footer-lists/footer-lists.js:71]: Why do we need `{' '}`?

File: src/containers/footer-lists/footer-lists.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+            {FOOTER_CATALOGS[language].title}
+          </Typography>
+        </div>
+        {categoriesList}
+      </div>
+      <div className={styles.cardBody}>
+        <div className={styles.cardTitle}>
+          <Typography variant='h5'>
+            {' '}
```
Comment (Alan18081) [inline src/containers/footer-lists/footer-lists.js:80]: Why do we need `{' '}`?

File: src/containers/footer-lists/footer-lists.js [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+            {FOOTER_INFORMATION[language].title}
+          </Typography>
+        </div>
+        {informationList}
+      </div>
+      <div className={styles.cardBody}>
+        <div className={styles.cardTitle}>
+          <Typography variant='h5'>
+            {' '}
```
Comment (Alan18081) [inline src/containers/footer-lists/footer-lists.js:89]: Why do we need `{' '}`?

File: src/containers/footer-lists/footer-lists.js [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+            {FOOTER_CONTACTS[language].title}
+          </Typography>
+        </div>
+        {contactsList}
+      </div>
+    </div>
+  );
+};
+
+const mapStateToProps = ({ Categories: { list } }) => ({
```
Comment (Alan18081) [inline src/containers/footer-lists/footer-lists.js:99]: Please, use hooks for redux

File: src/containers/footer-lists/footer-lists.js [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+const FooterLists = ({ getCategories, list, language = 0 }) => {
+  const styles = useStyles();
+
+  useEffect(() => {
+    getCategories();
+  }, [getCategories]);
+
+  const getCategoryURL = (category) => {
+    const [filteredCategory] = category.filter(
+      (item) => item.lang === URL_LANGUAGE
```
Comment (Alan18081) [inline src/containers/footer-lists/footer-lists.js:25]: Can `URL_LANGUAGE` be changed?

File: src/pages/home/category-item/category-item.js [hunk 11]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { useStyles } from './category-item.style';
+
+const Categories = ({ categoryName, categoryImage, categoryCode }) => {
+  const props = {
```
Comment (Alan18081) [inline src/pages/home/category-item/category-item.js:7]: Please, remove redundant variable `props`, just put actual object inside `useStyles`

File: src/pages/home/category-item/category-item.js [hunk 12]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { useStyles } from './category-item.style';
+
+const Categories = ({ categoryName, categoryImage, categoryCode }) => {
```
Comment (Alan18081) [inline src/pages/home/category-item/category-item.js:6]: Name of component should match with the name of file

File: src/pages/home/home-page/home.js [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+    ABOUT_US: 'ПРО НАС'
+  },
+  1: {
+    NEWS: 'NEWS',
+    ABOUT_US: 'ABOUT US'
+  }
+};
+
+const Home = () => {
+  const classes = useStyles();
```
Comment (Alan18081) [inline src/pages/home/home-page/home.js:22]: Please, rename to `styles`

File: src/pages/home/our-looks/our-looks.js [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+import { useStyles } from './our-looks.style';
+import {
+  LANGUAGE,
+  HOMEPAGE_TITLES,
+  HOMEPAGE_LOOKS_IMAGES
+} from '../../../configs';
+
+const OurLooks = () => {
+  const classes = useStyles();
```
Comment (Alan18081) [inline src/pages/home/our-looks/our-looks.js:12]: Please, rename to `styles`

File: src/containers/footer-links/footer-links.js [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Typography from '@material-ui/core/Typography';
+
+import { faInstagram, faFacebook } from '@fortawesome/free-brands-svg-icons';
+import { FOOTER_SOCIAL_NETWORK_LINKS } from '../../configs';
+
+import { useStyles } from './footer-links.styles';
+
+const FooterLinks = ({ language = 0 }) => {
```
Comment (Alan18081) [inline src/containers/footer-links/footer-links.js:11]: Don't use plain number for default value of parameter, use constant with explicit name

File: src/components/Cabinet/Cabinet.js [hunk 16]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (Alan18081) [inline src/components/Cabinet/Cabinet.js:1]: Use lowercased name for the folder and file

File: src/components/App-footer/app-footer.js [hunk 17]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (Alan18081) [inline src/components/App-footer/app-footer.js:1]: Please, lowercase the name of the folder.

File: src/components/LoadingBar/LoadingBar.js [hunk 18]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```
Comment (Alan18081) [inline src/components/LoadingBar/LoadingBar.js:1]: Please, rename folder and file from `LoadingBar` to `loading-bar` and use this format for other component files and folders

File: src/routes/Routes.js [hunk 19]
```diff
@@ -1,18 +1,26 @@
 import React from 'react';
```
Comment (Alan18081) [inline src/routes/Routes.js:1]: Please, lowercase the name of file

File: src/components/App-footer/app-footer.js (hunk 17 above)
Comment (Alan18081) [inline src/components/App-footer/app-footer.js:1]: Please, lowercase the name of the folder

File: src/components/Cabinet/Cabinet.js (hunk 16 above)
Comment (Alan18081) [inline src/components/Cabinet/Cabinet.js:1]: Please, lowercase the name of the file and the folder

File: src/components/LoggedCabinet/LoggedCabinet.js [hunk 20]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
Comment (Alan18081) [inline src/components/LoggedCabinet/LoggedCabinet.js:1]: Please, lowercase the name of the file and the folder

File: src/components/NotLoggedCabinet/NotLoggedCabinet.js [hunk 21]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (Alan18081) [inline src/components/NotLoggedCabinet/NotLoggedCabinet.js:1]: Please, lowercase the name of the file and the folder

File: .env.example (hunk 2 above)
Comment (YarkoSumyk) [reply .env.example:1]: Added .env.example

File: src/components/App-footer/app-footer.js (hunk 3 above)
Comment (YarkoSumyk) [reply src/components/App-footer/app-footer.js:9]: Resolved

File: src/components/Cabinet/Cabinet.js (hunk 4 above)
Comment (YarkoSumyk) [reply src/components/Cabinet/Cabinet.js:14]: Resolved

File: src/components/LoggedCabinet/LoggedCabinet.js (hunk 5 above)
Comment (YarkoSumyk) [reply src/components/LoggedCabinet/LoggedCabinet.js:18]: Resolved

File: src/containers/footer-lists/footer-lists.js (hunk 6 above)
Comment (YarkoSumyk) [reply src/containers/footer-lists/footer-lists.js:71]: Resolved

File: src/containers/footer-lists/footer-lists.js (hunk 7 above)
Comment (YarkoSumyk) [reply src/containers/footer-lists/footer-lists.js:80]: Resolved

File: src/containers/footer-lists/footer-lists.js (hunk 8 above)
Comment (YarkoSumyk) [reply src/containers/footer-lists/footer-lists.js:89]: Resolved

File: src/pages/home/home-page/home.js [hunk 22]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (Alan18081) [inline src/pages/home/home-page/home.js:1]: Please, rename file to the `home-page` to match with the name of the folder

File: src/redux/rootReduser.js [hunk 23]
```diff
@@ -1,9 +1,12 @@
 import { combineReducers } from 'redux';
```
Comment (Alan18081) [inline src/redux/rootReduser.js:1]: Rename `rootReduser` to `rootReducer`

File: src/components/App-footer/app-footer.js (hunk 17 above)
Comment (YarkoSumyk) [reply src/components/App-footer/app-footer.js:1]: Resolved

Comment (YarkoSumyk) [reply src/components/App-footer/app-footer.js:1]: Resolved

File: src/redux/rootReduser.js (hunk 23 above)
Comment (YarkoSumyk) [reply src/redux/rootReduser.js:1]: Renamed

File: src/pages/home/home-page/home.js (hunk 22 above)
Comment (YarkoSumyk) [reply src/pages/home/home-page/home.js]: Renamed

File: src/components/NotLoggedCabinet/NotLoggedCabinet.js (hunk 21 above)
Comment (YarkoSumyk) [reply src/components/NotLoggedCabinet/NotLoggedCabinet.js]: Renamed

File: src/components/LoggedCabinet/LoggedCabinet.js (hunk 20 above)
Comment (YarkoSumyk) [reply src/components/LoggedCabinet/LoggedCabinet.js]: Renamed

File: src/components/Cabinet/Cabinet.js (hunk 16 above)
Comment (YarkoSumyk) [reply src/components/Cabinet/Cabinet.js:1]: Renamed

File: src/routes/Routes.js (hunk 19 above)
Comment (YarkoSumyk) [reply src/routes/Routes.js:1]: Renamed

File: src/components/LoadingBar/LoadingBar.js (hunk 18 above)
Comment (YarkoSumyk) [reply src/components/LoadingBar/LoadingBar.js]: Renamed

File: src/components/Cabinet/Cabinet.js (hunk 16 above)
Comment (YarkoSumyk) [reply src/components/Cabinet/Cabinet.js:1]: Renamed

File: src/containers/footer-links/footer-links.js (hunk 15 above)
Comment (YarkoSumyk) [reply src/containers/footer-links/footer-links.js]: Resolved

File: src/containers/footer-lists/footer-lists.js (hunk 9 above)
Comment (YarkoSumyk) [reply src/containers/footer-lists/footer-lists.js]: Resolved

File: src/pages/home/our-looks/our-looks.js (hunk 14 above)
Comment (YarkoSumyk) [reply src/pages/home/our-looks/our-looks.js]: Renamed

File: src/pages/home/home-page/home.js (hunk 13 above)
Comment (YarkoSumyk) [reply src/pages/home/home-page/home.js]: Renamed

File: src/pages/home/category-item/category-item.js (hunk 12 above)
Comment (YarkoSumyk) [reply src/pages/home/category-item/category-item.js:6]: Renamed

File: src/containers/footer-lists/footer-lists.js (hunk 10 above)
Comment (YarkoSumyk) [reply src/containers/footer-lists/footer-lists.js:30]: Here is resolved this
https://github.com/horondi/horondi_client_fe/pull/53

File: src/pages/home/category-item/category-item.js (hunk 11 above)
Comment (YarkoSumyk) [reply src/pages/home/category-item/category-item.js:7]: It is resolved here https://github.com/horondi/horondi_client_fe/pull/54

File: .env (hunk 1 above)
Comment (YarkoSumyk) [reply .env]: Remved

**Pull request closed without merging**